general/ai/cli.go

```diff
@@ -40,47 +42,66 @@ func HowCmd(c *cli.Context) error {
 	if show, err := cliutils.ShowCmdHelpIfNeeded(c, c.Args()); show || err != nil {
 		return err
 	}
-	if c.NArg() < 1 {
+	if c.NArg() > 1 {
 		return cliutils.WrongNumberOfArgumentsHandler(c)
 	}
+	log.Output(coreutils.PrintTitle("This AI-based interface converts your natural language inputs into fully functional JFrog CLI commands.\n" +
+		"NOTE: This is a beta version and it supports mostly Artifactory and Xray commands.\n"))
 
-	args := cliutils.ExtractCommand(c)
-	question := questionBody{Question: fmt.Sprintf("How %s", strings.Join(args, " "))}
-	llmAnswer, err := askQuestion(question)
-	if err != nil {
-		return err
-	}
-	if strings.ToLower(llmAnswer) == "i dont know" {
-		log.Output("The current version of the AI model does not support this type of command yet.")
-		return nil
-	}
-	log.Output("AI generated JFrog CLI command:")
-	err = coreutils.PrintTable("", "", coreutils.PrintTitle(llmAnswer), false)
-	if err != nil {
-		return err
-	}
-
-	feedback := feedbackBody{questionBody: question, LlmAnswer: llmAnswer}
-	feedback.getUserFeedback()
-	if err = sendFeedback(feedback); err != nil {
-		return err
+	for {
+		var question string
+		scanner := bufio.NewScanner(os.Stdin)
+		fmt.Print("🐸 Your request: ")
+		for {
+			// Ask the user for a question
+			scanner.Scan()
+			question = scanner.Text()
+			if question != "" {
```

Comment:
Let's trim the spaces before this "if", to handle cases whereby the input is white spaces